src/application/notifications.ts

```diff
@@ -0,0 +1,56 @@
+import { User } from "../domain/user.js";
+import { Authentication, EmailClient } from "../infrastructure/email.js";
```

Comment:
how come the imports have explicit `.js` suffixes? is that required for ESM loading or something?

Reply:
Yeah the node runtime on Google Cloud complains if we don't have these.